.ci/validation.sh

```diff
@@ -1269,7 +1269,7 @@ openrewrite-recipes)
   set -e
   echo "Running OpenRewrite recipes..."
   ./mvnw -e --no-transfer-progress -Drewrite.recipeChangeLogLevel=INFO \
-  rewrite:run -P checkstyle-autofix
```

Comment:
no offence ment, want to keep things separated.
Interpreted the doc that its dedicated to check issues only, which is a sub scope to quality goal `sanity-check, therefore its included and lived there too. Also this goal is not activeByDefault doing no harm.

this is featured in the bigger composite recipe `CheckstyleImportLayout`.